### PR TITLE
Fix standalone docs in README

### PR DESCRIPTION
**Pull request by kelnos**
## Description

The README contains a brief note about running Mycroft standalone (without home.mycroft.ai), but the docs were very incomplete and somewhat outdated.

## How to test

n/a

## Contributor license agreement signed?
CLA [x] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

**Comment by devs-mycroft** [comment]
Hello, @kelnos, thank you for helping with the Mycroft project! We welcome everyone
into the community and greatly appreciate your help as we work to build an AI
for Everyone.

To protect yourself, the project, and users of Mycroft technologies we require
a Contributor Licensing Agreement (CLA) before accepting any code
contribution. This agreement makes it crystal clear that along with your
code you are offering a license to use it within the confines of this project.
You retain ownership of the code, this is just a license.

Please visit https://mycroft.ai/cla to initiate this one-time signing. Thank
you!

File: README.md [hunk 1]
```diff
@@ -84,14 +84,32 @@ When the configuration loader starts, it looks in these locations in this order,
...
+  "skills": {
+    "blacklisted_skills": [
+      "mycroft-configuration.mycroftai",
+      "mycroft-pairing.mycroftai"
+    ]
+  }
+}
+```
+
+Mycroft will then be unable to perform speech-to-text conversion, so you'll need to set that up as well, using one of the STT engines Mycroft supports.
```
Comment (forslund) [inline README.md:100]: Maybe we should link to the STT setup section of the documentation here: https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/stt-engine

Comment (kelnos) [reply README.md]: Good idea; updated

**Comment by forslund** [comment]
Thanks for the update! Looks good now. Merging.

**Pull request merged**